app/controllers/verify_controller.rb

```diff
@@ -3,5 +3,15 @@ class VerifyController < ApplicationController
 
   check_or_render_not_found -> { IdentityConfig.store.idv_api_enabled }, only: [:show]
 
-  def show; end
+  def show
+    @app_data = app_data
+  end
+
+  private
+
+  def app_data
+    {
+      initial_values: { personalKey: '0000-0000-0000-0000' },
```

Comment:
🙅 camel case in Ruby symbols, let's avoid it if we can? Or use strings if we really want the strings to match casing across Ruby/JS?
```suggestion
      initial_values: { 'personalKey' => '0000-0000-0000-0000' },
```

Reply:
I feel like there *has* to be a Rubocop for this, but `Naming/MethodName` seemed like the best option and `Naming/VariableNumber` didn't work either.. and `Style/SymbolLiteral` doesn't seem to have an option

Reply:
I'll probably switch to string here if that's agreeable enough. This won't stick around long-term, since eventually we don't want to be initializing anything for the client (client would receive personal key from response of API call to password confirmation step).

And yeah, couldn't find a Rubocop cop either! Would be nice to enforce this.

Reply:
yup, string is fine

Reply:
Updated to string key in b39c2dddb.